src/components/dialogs/create-case-dialog.js

```diff
@@ -84,8 +84,14 @@ export function CreateCaseDialog({ onClose, open }) {
             parentDirectoryUuid: activeDirectory,
         })
             .then()
-            .catch((message) => {
-                snackbarMessage(message, 'caseCreationError', { name });
+            .catch(() => {
```

Comment:
we are catching all errors here, this REST call could throw an error for other reasons than the file format
Shouldn't we display this error only if we catch the format error ?

Reply:
i agree

Reply:
ok